modules/configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery_{context}"]
+= Setting up a private storage endpoint on Microsoft Azure by configuring the Image Registry Operator to discover VNet and subnet names
+
+The following procedure shows you how to set up a private storage endpoint on Microsoft Azure by configuring the Image Registry Operator to discover VNet and subnet names.
+
+.Prerequisites 
+
+* You have configured the image registry to run on Azure.
+* Your network has been set up using the Installer Provisioned Infrastructure (IPI) installation method. 
```

Comment:
I don't think we are supposed to use the "IPI" abbreviation in docs, so I'd remove that